tests/conftest.py

```diff
@@ -127,7 +129,19 @@ def cells(bend90, bend180, bend90_euler, taper, straight) -> list[kf.KCell]:
 
 @pytest.fixture
 def pdk() -> kf.KCLayout:
-    kcl = kf.KCLayout("Test_PDK")
+    layerstack = kf.LayerStack(
+        wg=kf.kcell.LayerLevel(
+            layer=LAYER_CLASS.WG,
+            thickness=220,
+            zmin=0,
+            material="si",
+            info=kf.kcell.Info(mesh_order=1),
+        ),
+        clad=kf.kcell.LayerLevel(
+            layer=LAYER_CLASS.WGCLAD, thickness=3000, zmin=220, material="sio2"
+        ),
+    )
+    kcl = kf.KCLayout("Test_PDK", layer_stack=layerstack)
```

Comment:
This won't work, at least not the way you think it would work. You can't reference layers while the KCLayout isn't constructed yet, so anything using layers must be added post-creation.

Reply:
You can do this, but then it needs the logic to carry over layers from base_kcl to kcl and then apply the changes to the layerstack as well

Reply:
@sebastian-goeldi could you take a loo at this? Thanks